libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -131,7 +131,26 @@ const AP_Param::GroupInfo RangeFinder::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_ADDR", 23, RangeFinder, _address[0], 0),
 
-#if RANGEFINDER_MAX_INSTANCES > 1
+    // @Param: 1_POS_X
+    // @DisplayName:  X position offset
+    // @Description: X position of the first rangefinder in body frame. Use the zero range datum point if supplied.
+    // @Units: m
+    // @User: Advanced
+
+    // @Param: 1_POS_Y
+    // @DisplayName: Y position offset
+    // @Description: Y position of the first rangefinder in body frame. Use the zero range datum point if supplied.
+    // @Units: m
+    // @User: Advanced
+
+    // @Param: 1_POS_Z
+    // @DisplayName: Z position offset
+    // @Description: Z position of the first rangefinder in body frame. Use the zero range datum point if supplied.
+    // @Units: m
+    // @User: Advanced
+    AP_GROUPINFO("1_POS", 49, RangeFinder, _pos_offset[0], 0.0f),
+
+    #if RANGEFINDER_MAX_INSTANCES > 1
```

Comment:
Because it's easier to see. There was some discussion earlier in Gitter about indentation of preprocessor directives and there's no rules, but in the end it was basically decided:

- no spaces for first level of indentation
- spaces for the next indentation levels, either with leaving the # at the beginning or not